src/BuildProcess/ConfigureComposerAutoloader.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Laravel\VaporCli\BuildProcess;
+
+use Laravel\VaporCli\Helpers;
+use Symfony\Component\Finder\Finder;
+
+class ConfigureComposerAutoloader
+{
+    use ParticipatesInBuildProcess;
+
+    /**
+     * Execute the build process step.
+     *
+     * @return void
+     */
+    public function __invoke()
+    {
+        Helpers::step('<bright>Configuring Composer Autoloader</>');
```

Comment:
Is `</>` a typo? I saw this in a few places.

Reply:
Don't think so. It ends the symfony output formatting. `<bright>` is a custom style as per: https://github.com/laravel/vapor-cli/blob/f4ce5754f76f1b06d2538bebaf5f11277b2af526/src/Commands/Command.php#L84-L86

So using `</>` just resets the styles back to the default.